terragraph_planner/optimization/topology_interference.py

```diff
@@ -319,15 +326,25 @@ def _calculate_net_gain_on_rx_links(
 
             # Calculate net gain using the angle between the links
             tx_dev = angle_delta(
-                none_throws(tx_interfering_link.tx_dev),
                 none_throws(interfering_path.tx_dev),
+                none_throws(tx_interfering_link.tx_dev),
             )
             rx_dev = angle_delta(
-                none_throws(rx_interfered_link.rx_dev),
                 none_throws(interfering_path.rx_dev),
+                none_throws(rx_interfered_link.rx_dev),
+            )
+            tx_el_dev = angle_delta(
+                interfering_path.el_dev, tx_interfering_link.el_dev
+            )
+            rx_el_dev = -angle_delta(
+                interfering_path.el_dev, rx_interfered_link.el_dev
```

Comment:
I am confused about this one. Why do we add a minus sign here?

Reply:
Rx elevation deviation is minus tx elevation deviation. We don't store both of these values.

Reply:
Ah I see. el_dev is only tx so no matter what links are, it is always tx elevation deviation, is it correct?

Reply:
Exactly